src/game/engine/techno.cpp

```diff
@@ -837,47 +1116,48 @@ VisualType TechnoClass::Visual_Character(BOOL flag) const
         return VISUAL_NORMAL;
     }
 
-    if (!Techno_Class_Of().Is_Invisible() || m_PlayerOwned || g_InMapEditor) {
```

Comment:
I assume this has been properly tested? In general I don't agree with rewriting code that already works, its just asking to accidentally introduce bugs.